charts/shopware/values.yaml

```diff
@@ -176,6 +176,19 @@ store:
   #     business_event_handler_buffer:
   #       level: info
 
+  opensearch:
+    enabled: true
+    username: shopware
+    passwordSecrfetRef:
+      name: opensearch-credential
+      key: password
+    host: cluster-1.opensearch.svc.cluster.local
```

Comment:
Only missing is port, then this can be added to the store template resource and passed one by one to the operator crd :+1: